code/modules/power/apc.dm

```diff
@@ -783,66 +783,63 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/power/apc/auto_name, APC_PIXEL_OFFSET
 		return
 	if(!ishuman(user))
 		return
+	var/mob/living/carbon/human/apc_interactor = user
+	var/obj/item/organ/stomach/maybe_stomach = apc_interactor.getorganslot(ORGAN_SLOT_STOMACH)
+	if(!istype(maybe_stomach, /obj/item/organ/stomach/ethereal))
+		togglelock(user)
```

Comment:
```suggestion
	var/obj/item/organ/stomach/ethereal/maybe_stomach = apc_interactor.getorganslot(ORGAN_SLOT_STOMACH)
	if(!istype(maybe_ethereal_stomach))
		togglelock(user)
```
We only care about the stomach if it's an ethereal stomach type specifically, and `istype(thing, /path/we/want)` already returns false if `thing` is null, so we don't need the intermittent cast to a normal stomach